fluentnamer/readme.md

```diff
@@ -19,7 +19,12 @@ pipeline:
     flatten-models: true
     flatten-payloads: true
     naming:
-      preserve-uppercase-max-length: 4
+      override:
+        api: Api
+        cpk: Cpk
+        sas: Sas
+        url: Url
+        xml: Xml
```

Comment:
`SAS, XML, HTTP, JSON, API, URL, CPK`, it seems there is `Http` and `Json`

Reply:
I've deleted the `preserve-uppercase-max-length: 4` line. So all 4 letter will be changed.